script/api.py

```diff
@@ -20,6 +35,13 @@
 conn.autocommit = True
 
 
+@app.route("/IsloggedIn", methods=["POST"])
```

Comment:
```suggestion
@app.route("/is_logged_in", methods=["POST"])
```
We use snake case in python.